JS_lesson5/Task2.js

```diff
@@ -0,0 +1,90 @@
+// 1. Написать скрипт, переводящий количество байт в нужные единицы
+//   bytes => kB Mb Gb Tb
+//   16 565 846 bytes (16,6 Mb)
+
+//   1 Kb = 1024 byte
+//   1 Mb = 1024 Kb
+//   1 Gb = 1024 Mb
+//   1 Tb = 1024 Gb
+
+//   // Пример: ~ 1000
+//   4 548 = 4,5 Kb (Real 4,4 Kb)
+//   454 548 = 454,5 Kb
+//   1 454 548 = 1,5 Mb
+
+//   Результат должен быть округлен до 1 знака после запятой методом .toFixed(), про который надо почитать самим ;)
+
+const byte = 10000000000000;
+let size = '';
+let result = byte;
+let i = 0
+while (true) {
+    if (result / 1024 >= 1) {
+        i++;
+        result = result / 1024
+    } else {
+        break;
+    }
+}
+switch (i) {
+    case 0: 
+        size = 'byte';
+        break;
+    case 1: 
+        size = 'Kb'
+        break;
+    case 2: 
+        size = 'Mb'
+        break;
+    case 3:
+        size = 'Gb'
+        break;
+    case 4:
+        size = 'Tb'
+        break; 
+}   
+console.log(`${byte} = ${result.toFixed(1)} ${size}`);
+
+
+// 2. Сделать из "*" в консоли равнобедренный треугольник и ромб
```

Comment:
2 in 1 ? :)
Need left side of diamond )
![image](https://github.com/AlexandrBelan/JS_HW/assets/103571439/b9036a6c-085f-4343-9c44-fa84ddd97f12)